src/Http/Response.php

```diff
@@ -310,4 +310,17 @@ private function checkHeaders()
 		}
 	}
 
+
+	/**
+	 * @return string
+	 */
+	public function getNonce()
+	{
+		static $nonce;
```

Comment:
Private property is better. This makes the nonce shared across different Response instances.